src/itemlistformaction.cpp

```diff
@@ -131,12 +131,13 @@ void ItemListFormAction::process_operation(Operation op,
 			itemposname);
 		if (itemposname.length() > 0 && visible_items.size() != 0) {
 			if (itempos < visible_items.size()) {
+				if (v->open_in_browser(visible_items[itempos].first->link())) {
+					break;
```

Comment:
From a user's perspective, it'd be nice if this showed an error as well; ideally including browser's exit code. Otherwise user might be left with a feeling that their keypress was not processed at all. (The error messages can be shown with `v->show_error(_("Message here"))`).